src/WebServer/Http/HttpServer.cs

```diff
@@ -66,14 +68,28 @@ public void StopServer()
                     using (var inputStream = args.Socket.InputStream)
                     {
                         var request = await MutableHttpServerRequest.Parse(inputStream);
+                        if (!request.IsComplete)
+                        {
+                            _log.Warn($"Request {request.Method} {request.Uri} aborted.");
+                            return;
+                        }
+
+                        if (_log.IsDebugEnabled())

```

Comment:
The debug method itself should have it as well, the only thing the _log.IsDebugEnabled() checks will make sure that the string won't be computed. Since these are in the critical path for request handling I didn't want the strings to be computed.